modules/coremark/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT CONFIG_COREMARK)
+  return()
+endif()
```

Comment:
I think we don't need this line as the CMake is added conditionally in the directory above:

```add_subdirectory_ifdef(CONFIG_COREMARK coremark)```